GameFighterXUnitTests/GameFighterTests.cs

```diff
@@ -0,0 +1,185 @@
+using GameFighter;
+using Xunit;
+
+namespace GameFighterXUnitTests
+{
+    public class GameFighterTests
+    {
+        [Fact]
+        public void Health_Warrior()
+        {
+            var warrior = new Warrior();
+            Assert.Equal(50, warrior.Health);
+        }
+
+        [Fact]
+        public void Health_Knight()
+        {
+            var knight = new Knight();
+            Assert.Equal(50, knight.Health);
+        }
+
+        [Fact]
+        public void Attack_Warrior()
+        {
+            var warrior = new Warrior();
+            Assert.Equal(5, warrior.Attack);
+        }
+
+        [Fact]
+        public void Attack_Knight()
+        {
```

Comment:
This test and the tests above don't test anything. You don't need to check how compiler works